recipe/meta.yaml

```
@@ -20,7 +20,7 @@ requirements:
     - {{ compiler('c') }}
     - ninja
```

```suggestion
    - ninja-base
```
because the package doesn't rely on python